stream-chat-android-ui-common/src/test/kotlin/com/getstream/sdk/chat/Mother.kt

```diff
@@ -0,0 +1,167 @@
+package com.getstream.sdk.chat
```

Comment:
This code is unfortunately duplicated between the old UI module and the common module for now

Reply:
Why not to put it to common test module?